test-project/Packages/manifest.json

```diff
@@ -5,7 +5,8 @@
     "com.improbable.gdk.dependencytest": "file:com.improbable.gdk.dependencytest",
     "com.improbable.gdk.querybasedinteresthelper": "file:../../workers/unity/Packages/com.improbable.gdk.querybasedinteresthelper",
     "com.improbable.gdk.testutils": "file:../../workers/unity/Packages/com.improbable.gdk.testutils",
-    "com.improbable.gdk.tools": "file:../../workers/unity/Packages/com.improbable.gdk.tools"
+    "com.improbable.gdk.tools": "file:../../workers/unity/Packages/com.improbable.gdk.tools",
+    "com.unity.modules.physics": "1.0.0"
```

Comment:
Entities now requires the physics module since it uses `UnityEngine.Collider`.
Won't compile without. Bit odd that the package itself doesn't have this as a required module however

Reply:
that's because built-in packages behave slightly different in the dependency tree